buf/registry/module/v1beta1/branch.proto

```diff
@@ -60,13 +60,13 @@ message Branch {
   // Only one Branch per module will be marked as the release Branch.
   // TODO: enum?
   bool is_release = 9 [(buf.validate.field).required = true];
-  // The id of the latest Commit created on the Branch.
-  string latest_commit_id = 10 [
+  // The id of the head Commit created on the Branch.
+  string head_commit_id = 10 [
     (buf.validate.field).required = true,
```

Comment:
I noticed this is required. Does this imply that an empty Branch does not exist, or should we relax this constraint?

Reply:
Unless this causes a problem elsewhere, it makes sense to me that an empty branch doesn't exist because this is true in Git (a branch pointer has to point to a commit).

Reply:
I updated the docs to reflect this.